includes/functions/core.php

```diff
@@ -0,0 +1,298 @@
+<?php
+/**
+ * The core namespace for the Dynamic CDN project
+ *
+ * @author Eric Mann <eric@eamann.com>
+ */
+
+namespace EAMann\Dynamic_CDN\Core;
+
+use EAMann\Dynamic_CDN\DomainManager;
+
+// @todo if this was a class, we could self contain this variable. It's needed in multiple functions so we need to put it in the namespaced global scope
+$context = 'uploads';
```

Comment:
i think we should name this variable something else if it's going to be a global. like with a prefix.